pkgs/games/minecraft/default.nix

```diff
@@ -151,6 +151,7 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
     platforms = [ "x86_64-linux" ];
+    broken = builtins.trace "Warning: minecraft-launcher will fail on NixOS for versions >1.19, try prismlauncher or atlauncher instead" false;
```

Comment:
given that this has been [broken since June 2022](https://github.com/NixOS/nixpkgs/issues/179323), i would strongly encourage this package to be marked as broken instead of simply warning.

Reply:
changed to `true`.